metador_core/widget/dashboard.py

```diff
@@ -303,15 +303,19 @@ def _resolve_node(self, node: MetadorNode, wmeta: WidgetConf) -> WidgetConf:
 
     def show(self) -> Viewable:
```

Comment:
Please refactor the method, it is doing way too much.

* Keep the logical steps (creating rows, adding widgets to rows) from the technical details (creating the `FlexBox`es with lots of parameters), create some helper methods for that

* avoid duplication, such as e.g. the font style. Create a dict `font_style` with the font style and pass it as arguments using  `**font_style`.